packages/api/src/modules/notify/@types/NotifyOutPipe.ts

```diff
@@ -0,0 +1,18 @@
+import { NotificationDataType } from "./NotificationDataType";
+import { NotificationType } from "./NotificationType";
+
+export interface NotificationValidationType {
```

Comment:
J'ai mis du temps à comprendre ce que c'était avec ce nom-là. Peut-être plutôt 

```suggestion
export interface NotificationOutcomeType {
```
ou

```suggestion
export interface NotificationResultType {
```

?

Reply:
Je croyais que mes deux propositions collaient mais je ne comprends pas ton explication ><

Reply:
Ah je comprends maintenant. Je dirais `NotifierType` ou `NotifierMethodType` mais même si ton objectif c'est de la validation, l'objet n'est pas de type validation donc ça embrouille si ça s'appelle Validation